packages/bff/src/services/selfcareService.ts

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+
+import { UserId } from "pagopa-interop-models";
+import {
+  InstitutionResource,
+  ProductResource,
+  SelfcareV2Client,
+  UserResource,
+} from "pagopa-interop-selfcare-v2-client";
+import { userNotFound } from "../model/domain/errors.js";
+
+export function selfcareServiceBuilder(selfcareV2Client: SelfcareV2Client) {
+  return {
+    async getSelfcareUser(
+      userId: UserId,
+      userIdQuery: string,
+      institutionId: string
+    ): Promise<UserResource> {
+      const users = await selfcareV2Client.getInstitutionProductUsersUsingGET({
+        params: { institutionId },
+        queries: {
+          userIdForAuth: userId,
+          userId: userIdQuery,
+        },
+      });
+
+      const user = users.find((u) => u.id === userIdQuery);
```

Comment:
I think it is the same but It seems the [scala code](https://github.com/pagopa/interop-be-backend-for-frontend/blob/1.0.x/src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/SelfcareApiServiceImpl.scala#L100) takes the first one without find.

Reply:
Yes, i can probably use `.at(0)` in order for it to be equivalent to Scala code